presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -176,6 +177,7 @@ void PrestoServer::run() {
   }
 
   registerPrestoCppCounters();
+  registerVeloxCounters();
```

Comment:
Shall we simply combine registerPrestoCppCounters and registerVeloxCounters into one 
```
registerStatsCounters()
```